recipes-core/images/full.bb

```diff
@@ -15,6 +15,7 @@ BISDN_SWITCH_IMAGE_EXTRA_INSTALL += "\
     file \
     frr \
     git \
+    grpc-cli \
```

Comment:
I realize that there a many commits in the PR already, but this commit would probably have benefited from being split into multiple commits. At least as far as reviewing is concerned.

Reply:
There really isn't a way to split this into multiple commits, as there were no intermediate versions. The changes were recreated from scratch on top of the new grpc vesion.